### PR TITLE
[rtwcli] rtw cli base package with dummy info command

### PR DESCRIPTION
**Pull request by muritane**

### CODE REVIEW

**Review by destogl — CHANGES_REQUESTED** [review]
Just few things to clarify!

File: rtwcli/rtwcli/rtwcli/command/extension_points.py
[the remark this answers is not in the document]
Comment (muritane) [reply rtwcli/rtwcli/rtwcli/command/extension_points.py:1]: `~$ rtw extensions` What extensions are available?
```
rtw_cmds.docker.verbs
  enter: Enter docker container
rtw_cmds.pkg.verbs
  create: Create a new ROS package
rtwcli.command
  docker: Various Docker related sub-commands
  extension_points: List extension points
  extensions: List extensions
  info: Provide useful information
  pkg: Various package related sub-commands
```

`~$ rtw extension_points` What extension _interfaces_ are available?
```
rtw_cmds.docker.verbs: The interface for verb extensions
rtw_cmds.pkg.verbs: The interface for verb extensions
rtwcli.command: The extension point for 'command' extensions
```
For our case "extension_points" are not really necessary as we use the base verb from rtwcli.VerbExtension and base command from rtw_cmds.BaseCommmand. Should I remove extension_points?

File: rtwcli/rtwcli/rtwcli/command/__init__.py [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    :param str dest: name of the attribute under which the selected extension
+      will be stored
+    :param dict command_extensions: dict of command extensions by their name
+      where each contributes a command with specific arguments
+    """
+    import warnings
+
+    warnings.warn(
+        "'rtwcli.command.add_subparsers' is deprecated, use "
+        "'rtwcli.command.add_subparsers_on_demand' instead",
```
[the remark this answers is not in the document]
Comment (muritane) [reply rtwcli/rtwcli/rtwcli/command/__init__.py]: removed the deprecated method

File: rtwcli/rtwcli/rtwcli/command/__init__.py [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+from rtwcli.entry_points import get_entry_points
+from rtwcli.entry_points import get_first_line_doc
+from rtwcli.plugin_system import instantiate_extensions
+from rtwcli.plugin_system import PLUGIN_SYSTEM_VERSION
+from rtwcli.plugin_system import satisfies_version
+
+
+class CommandExtension:
+    """
+    The extension point for 'command' extensions.
```
Comment (destogl) [inline rtwcli/rtwcli/rtwcli/command/__init__.py:31]:
```suggestion
    The extension point for 'command' extensions. Those are the base extensions in RTW.
```

**Review by destogl — DISMISSED** [review]
looks great!

**Review by destogl — APPROVED** [review]

**Pull request merged**